Kickstarter-iOS/Views/Storyboards/Main.storyboard

```diff
@@ -1,7 +1,11 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="10117" systemVersion="15E65" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" initialViewController="49e-Tb-3d3">
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
i was getting some build errors due to bad xibs. all i had to do was open the xib and close it to fix. it may have been just an intermittent thing. i might undo these changes and try again...